### ISSUE TITLE
Create react app is broken because of 7.9.0

### ISSUE THREAD

**Issue by Lukavyi** [babel/babel#11292]
https://github.com/facebook/create-react-app/issues/8680

# Describe the bug
Running npm run build will fail on any project, even on a vanilla app created with create-react-app will fail with the following error:

```
Error: [BABEL] /Users/<username>/Projects/my-app/src/index.js: Could not find plugin "proposal-numeric-separator". Ensure there is an entry in ./available-plugins.js for it. (While processing: "/Users/<username>/Projects/my-app/node_modules/babel-preset-react-app/index.js$0")
    at Array.map (<anonymous>)
    at Generator.next (<anonymous>)
    at Generator.next (<anonymous>)


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my-app@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the my-app@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by existentialism** [comment]
@Lukavyi please read my comment within the thread: https://github.com/facebook/create-react-app/issues/8680#issuecomment-601820935

**Comment by ffrezende** [comment]
My pipelines broke, I have the same error. Does someone have a solution for this bug?

**Comment by nicolo-ribaudo** [comment]
Try adding `"@babel/compat-data": "~7.8.0"` to your `"devDependencies"`, deleting `"@babel/compat-data"` from `package-lock.json` and running `npm install`.

**Comment by Fedeorlandau** [comment]
Same here, pipeline is broken

**Comment by wandersoncesar** [comment]
> Try adding `"@babel/compat-data": "~7.8.0"` to your `"devDependencies"`, deleting `"@babel/compat-data"` from `package-lock.json` and running `npm install`.
> 
> If you are using yarn, this is better: [facebook/create-react-app#8680 (comment)](https://github.com/facebook/create-react-app/issues/8680#issuecomment-601851525)

Works for me in my **Angular** application. Thanks! 

I just add `"@babel/compat-data": "~7.8.0"` to my `"devDependencies"`and ran my pipeline again.

**Comment by nicolo-ribaudo** [comment]
@wandersoncesar It's unexpected that it broke angular 🤔
You just need to avoid `@babel/preset-env` v7.8.0-7.8.6 which are broken.

**Comment by ChrisEddy** [comment]
Our pipeline is broken because of this. Can anyone recommend a react-scripts version to roll back to that works?

**Comment by Lukavyi** [comment]
> Our pipeline is broken because of this. Can anyone recommend a react-scripts version to roll back to that works?


https://github.com/facebook/create-react-app/issues/8680#issuecomment-601851525

**Comment by ChrisEddy** [comment]
> > Our pipeline is broken because of this. Can anyone recommend a react-scripts version to roll back to that works?
> 
> [facebook/create-react-app#8680 (comment)](https://github.com/facebook/create-react-app/issues/8680#issuecomment-601851525)

Thank you!!

**Comment by jdmarshall** [comment]
Yeah I have a whole team staying late on a Friday afternoon for this.

Why does @babel/core have caret imports of all of its children?  We shouldn't be picking up every child dependency upgrade if we are still using babel/core@7.2.2

**Comment by nicolo-ribaudo** [comment]
> Why does @babel/core have caret imports of all of its children?

Because that's almost the standard for open source JS projects?

Anyway, I'm really surprised that `@babel/core`'s dependencies have been upgraded, both `npm` and `yarn` prevent it (even when `^` is used) thanks to their lockfiles :thinking:

**Comment by jalanya** [comment]
Locally in my macOS Catalina 10.15.0, I'm getting the following error:
```
Failed to compile.

./src/apps/common/hoc/model/index.js
ReferenceError: Unknown helper createSuper
```
my configuration is
package.json
```
{
  ...
   "devDependencies": {
    "@babel/preset-env": "^7.5.5",
    "@babel/preset-react": "^7.0.0",
    "npm-force-resolutions": "0.0.3",
    ...
    ..
   },
  "scripts": {
    "preinstall": "npx npm-force-resolutions",
     ...
     ...
   } 
   "resolutions": {
    "@babel/preset-env": "^7.8.7"
  }
}
```

And from jenkins, also, i'm getting the following error:

```
17:31:44  > npx npm-force-resolutions
17:31:44  
17:31:48  npx: installed 5 in 3.326s
17:31:48  ENOENT: no such file or directory, open './package-lock.json'
17:31:49  npm ERR! code ELIFECYCLE
17:31:49  npm ERR! errno 1
```
Do you know if three is a workaround for both cases? Thanks

**Comment by nicolo-ribaudo** [comment]
I don't know about the second error, but they are releasing a new CRA version with the fix!

**Comment by nicolo-ribaudo** [comment]
Fixed in [create-react-app 3.4.1](https://github.com/facebook/create-react-app/releases/tag/v3.4.1). It should also make the error about "createSuper" disappear (even if it's an actual bug in Babel).

**Comment by rockus123** [comment]
Modern development environment with babel, react and other dependencies will create havoc all the times.  Changes break suddenly and errors are not clear.

**Comment by kaicataldo** [comment]
Lockfiles were introduced to help mitigate the problem of shifting dependencies. Highly recommend using one!

**Comment by tatimatch** [comment]
this(npm i @babel/compat-data@7.8.0) works for me! thanks

**Comment by jblevins1991** [comment]
You should not be using the caret symbol if you are going to accept breaking changes into minor releases on code that you control. If you introduce breaking changes on a minor version release you should be burdened with fixing it, not everyone that depends on you. Why is this such a hard concept in the javascript industry.

**Comment by nicolo-ribaudo** [comment]
@jblevins1991 We fixed it in 7.8.7.

If you depend on open source software you are not helping to maintain and you are not supporting in any way (neither with human time nor financially), you are not entitled to complain about what the maintainers are doing. I assure you that we are trying to do our best. Why is this such a hard concept in the tech industry.
